packages/suite-desktop/package.json

```diff
@@ -82,9 +82,7 @@
             "oneClick": false
         },
         "mac": {
-            "files": [
-                "entitlements.mac.inherit.plist"
-            ],
+            "files": ["entitlements.mac.inherit.plist"],
```

Comment:
Please check your code formatter and revert format changes.
We use Prettier and Prettier puts it on more lines. If we merge it like this, the next person who will edit this file will change it back.

Reply:
ca75f33